node/src/components/network/tests_bulk_gossip.rs

```diff
@@ -127,7 +129,31 @@ impl Display for DummyPayload {
     }
 }
 
+/// Reads an envvar from the environment and, if present, parses it.
+///
+/// Only absent envvars are returned as `None`.
+///
+/// # Panics
+///
+/// Panics on any parse error.
+fn read_env<T: FromStr>(name: &str) -> Option<T>
+where
+    <T as FromStr>::Err: Debug,
+{
+    match env::var(name) {
+        Ok(raw) => Some(
+            raw.parse()
+                .unwrap_or_else(|_| panic!("cannot parse envvar `{}`", name)),
+        ),
+        Err(env::VarError::NotPresent) => None,
+        Err(err) => {
+            panic!(err)
+        }
+    }
+}
```

Comment:
This method is useful enough to have it in some `utils` package.

Reply:
Can do! :)